guides/security/BYOD.md

```diff
@@ -0,0 +1,101 @@
+# Bring Your Own Device Policy
+
+This document highlights things you should think about, and how you manage your own device while working on Made Tech products and services. Not only the data you work with but also the software you have installed and how you protect it - like antivirus software.
+
+This only applies to you if you’re using your own device for Made Tech work - including checking your Made Tech email. Devices aren’t just desktop and laptops - they include, for example, tablets and smartphones.
+
+## Using and managing your device
+
+If you’re using your own device for Made Tech work, it means you’re being trusted to use Made Tech information appropriately and keep any Made Tech information or data you hold, or can access, safe.
```

Comment:
Feel like we can trim the last sentence without losing meaning to: and keep any Made Tech information safe